feature/dashboard/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
You can remove "dashboard", as each String file is individual it is not necessary to make more specifications

Reply:
Comment solved